test/Microsoft.Azure.Devices.Edge.Test/EdgeAgentDirectMethods.cs

```diff
@@ -143,27 +142,81 @@ public async Task TestUploadModuleLogs()
                 }, token);
             await Task.Delay(10000);
 
-            var request = new ModuleLogsUploadRequest("1.0", new List<LogRequestItem> { new LogRequestItem(moduleName, new ModuleLogFilter(Option.None<int>(), Option.None<string>(), Option.None<string>(), Option.None<int>(), Option.None<string>())) }, LogsContentEncoding.None, LogsContentType.Text, sasUrl);
+            var request = new
+            {
+                schemaVersion = "1.0",
+                items = new
+                {
+                    id = "NumberLogger",
+                    filter = new { },
+                },
+                encoding = 0,
+                contentYtpe = 1,
+                sasUrl = Context.Current.BlobSasUrl,
```

Comment:
`Context.Current.BlobSasUrl` is an `Option<string>`, so will this work? I think you want to `Expect` it, like you did with `Context.Current.NumberLoggerImage` a little farther down in the next test.

Reply:
you're right